Popular Movies/Popular Movies/Model/Date.h

```diff
@@ -0,0 +1,15 @@
+//
+//  Date.h
+//  Popular Movies
+//
+//  Created by Test on 14/10/2017.
+//  Copyright © 2017 Test. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
```

Comment:
Is there a specific reason why do we need this class? If we need to use part of the date (e.g. day or month) we can use **[NSDateComponents](https://developer.apple.com/documentation/foundation/nsdatecomponents)**.